index.html

```diff
@@ -95,6 +95,7 @@
 											<li><a target="_self" href="https://query-server.herokuapp.com/">Query Server</a></li>
 											<li><a target="_self" href="http://knitting.fossasia.org/">Knitting Lab</a></li>
 											<li><a target="_self" href="https://badgeyay-dev.herokuapp.com/">Badgeyay</a></li>
+							     				<li><a target="_self" href="https://github.com/fossasia/labyrinth">Labyrinth</a></li>
```

Comment:
Pls wait till https://github.com/fossasia/labyrinth/issues/38 is done so that you can add the deployment link

Reply:
ohk ;D